Documentation Standard Updates (#25)
fivetran/dbt_shopify_holistic_reporting: README.md

This package builds off of the [Shopify dbt package](https://github.com/fivetran/dbt_shopify) to weave together your Shopify e-commerce data with insights from marketing connectors. Currently, this package supports combining Shopify with email and SMS marketing data from Fivetran's [Klaviyo dbt package](https://github.com/fivetran/dbt_klaviyo).
> Wanna see Shopify combined with another connector? Please make create a [Feature Request](https://github.com/fivetran/dbt_shopify_holistic_reporting/issues/new?assignees=&labels=enhancement&template=feature-request.yml&title=%5BFeature%5D+%3Ctitle%3E).

This dbt package enables you to:
- Tie e-commerce revenue to your email and SMS marketing via last-touch attribution.
- Consolidate customers, their information, and activity across platforms.
- Create a rich portrait of customer personas based on how customers are engaging with and responding to specific marketing efforts.

Check out our [blog post](https://www.fivetran.com/blog/gain-faster-insights-from-shopify-and-klaviyo-data) for further discussion on how the package can accelerate your business analysis.

<!--section="shopify_holistic_reporting_transformation_model"-->

> The following table provides a detailed list of all models materialized within this package (see [Shopify](https://github.com/fivetran/dbt_shopify#-what-does-this-dbt-package-do) and [Klaviyo](https://github.com/fivetran/dbt_klaviyo#-what-does-this-dbt-package-do) for the upstream models these are built off of).
> > TIP: See more details about these models in the package's [dbt docs site](https://fivetran.github.io/dbt_shopify/#!/overview/shopify_holistic_reporting).
> 
> | **model**                | **description**                                                                                                                                |

Author: ```suggestion
The following table provides a detailed list of all tables materialized within this package (see [Shopify](https://github.com/fivetran/dbt_shopify#-what-does-this-dbt-package-do) and [Klaviyo](https://github.com/fivetran/dbt_klaviyo#-what-does-this-dbt-package-do) for the upstream tables these are built off of).
> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_shopify/#!/overview/shopify_holistic_reporting).

| **Table**                | **Description**                                                                                                                                |
```

> TIP: See more details about these tables in the package's [dbt docs site](https://fivetran.github.io/dbt_shopify/#!/overview/shopify_holistic_reporting).

| **Table**                | **Description**                                                                                                                                |
| ------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------- |
| [shopify_holistic_reporting__orders_attribution](https://fivetran.github.io/dbt_shopify_holistic_reporting/#!/model/model.shopify_holistic_reporting.shopify_holistic_reporting__orders_attribution)             | Each record represents a unique Shopify order, enhanced with a customizable last-touch attribution model associating orders with Klaviyo flows and campaigns that customers interacted with. Includes dimensions like whether it is a new or repeat purchase in Shopify. See available customizations [here](https://github.com/fivetran/dbt_klaviyo#attribution-lookback-window). Materialized incrementally by default. |

> | [shopify_holistic_reporting__daily_customer_metrics](https://fivetran.github.io/dbt_shopify_holistic_reporting/#!/model/model.shopify_holistic_reporting.shopify_holistic_reporting__daily_customer_metrics)             | Each record represent a unique customer's daily activity attributed to a campaign or flow in Klaviyo. The grain is set at the customer-day-flow/campaign level. This model is enriched with both Shopify and Klaviyo metrics, such as the net revenue, taxes paid, discounts applied, and the counts of each type of interaction between the user and the campaign/flow. |

Reviewer: ```suggestion
| [shopify_holistic_reporting__daily_customer_metrics](https://fivetran.github.io/dbt_shopify_holistic_reporting/#!/model/model.shopify_holistic_reporting.shopify_holistic_reporting__daily_customer_metrics)             | Each record represent a unique customer's daily activity attributed to a campaign or flow in Klaviyo. The grain is set at the customer-day-flow/campaign level. This table is enriched with both Shopify and Klaviyo metrics, such as the net revenue, taxes paid, discounts applied, and the counts of each type of interaction between the user and the campaign/flow. |
```

    klaviyo_union_databases: ['klaviyo_usa','klaviyo_canada'] # use this if the data is in different databases/projects but uses the same schema name
```

### Step 4: Set Shopify- and Klaviyo-specific configurations
See connector-specific configurations in their individual dbt package READMEs:
- [Shopify](https://github.com/fivetran/dbt_shopify)
- [Klaviyo](https://github.com/fivetran/dbt_klaviyo)

### (Optional) Step 5: Additional configurations

> <details><summary>Expand for configurations</summary>

Reviewer: ```suggestion
<details open><summary>Expand/Collapse details</summary>
```
